routers/crudRouter.js

```diff
@@ -2,6 +2,9 @@
 const express = require('express')
 const MongoClient = require('mongodb').MongoClient
 const MsLogger = require('@first-lego-league/ms-logger').Logger()
+const { authroizationMiddlware } = require('@first-lego-league/ms-auth')
+
+const adminAction = authroizationMiddlware(['admin', 'scorekeeper', 'development'])
```

Comment:
Notice that in the scoring app i allowed the scorekeeper to do anything that the admin could do. I don't think this rule should be in effect here. I think that `adminAction` shuold be only performed by `admin` or `development`